File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/save-courtesy-batch_param.ts


import { ISaveCourtesyBatch_Param } from './isave-courtesy-batch_param';

export class SaveCourtesyBatch_Param implements ISaveCourtesyBatch_Param {
  
	public CourtesyBatchID?: number;
	public VettingBatchID?: number;
	public VettingTypeID?: number;
	public VettingBatchNotes: string = "";
	public AssignedToAppUserID?: number;
	public ResultsSubmittedDate?: Date;
	public ResultsSubmittedByAppUserID?: number;
	public isSubmit: boolean = false;
  
}

File: GTTS/INL.VettingService.Models/SaveCourtesyBatch_Param.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class SaveCourtesyBatch_Param : ISaveCourtesyBatch_Param
    {
        public long? CourtesyBatchID { get; set; }
        public long? VettingBatchID { get; set; }
        public int? VettingTypeID { get; set; }
        public string VettingBatchNotes { get; set; }
        public int? AssignedToAppUserID { get; set; }
        public DateTime? ResultsSubmittedDate { get; set; }
        public int? ResultsSubmittedByAppUserID { get; set; }
        public int? ModifiedByAppUserID { get; set; }
        public bool isSubmit { get; set; }
    }
}
